_data/provincias/burgos.json

```
@@ -2,6 +2,11 @@
   "name": "Burgos",
   "comunidad": "castilla-y-leon",
   "webs": [
+    {
+      "url": "http://www.adradadehaza.es",
```

El dominio tiene que ir sin protocolo, pero creo que acabo de añadir este mismo ayuntamiento en otro PR tuyo, ¿puede ser?
```suggestion
      "url": "www.adradadehaza.es",
```

---

si, se debieron mezclar los PRs.